src/Http/Productions/ProductionsSearchController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace CultuurNet\UDB3\Http\Productions;
+
+use CultuurNet\UDB3\Event\Productions\Production;
+use CultuurNet\UDB3\Event\Productions\ProductionRepository;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProductionsSearchController
+{
+    /**
+     * @var ProductionRepository
+     */
+    private $repository;
+
+    public function __construct(ProductionRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function search(Request $request): Response
```

Comment:
I was thinking we should probably add pagination to this request to be safe?

If we do, I'd follow the `start` and `limit` parameters we already use in various places like [Search API3](https://github.com/cultuurnet/udb3-search-service/blob/master/src/Http/OfferSearchController.php#L143-L144) and the [labels query endpoint](https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/Label/Query/QueryFactory.php#L14-L15) for consistency.

For the response data, you can use the existing [`CultuurNet\Hydra\PagedCollection` class](https://github.com/cultuurnet/Hydra/blob/master/src/PagedCollection.php). For an example of how to use it, you can look at [`CultuurNet\UDB3\Http\Label::createPagedCollection()`](https://github.com/cultuurnet/udb3-silex/blob/master/src/Http/Label/ReadRestController.php#L101-L125) (a private method and specific to that controller, but a decent example). That way we maintain a consistent response format for paginated requests based on a w3 standard.

If you have any more questions how to implement this, let me know!